#! /bin/sh /usr/share/dpatch/dpatch-run
## 34-bts386713.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: load libraries with their complete name.

@DPATCH@
--- ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/PortableGroup/PortableGroup_ORBInitializer.cpp	2007-01-25 18:04:11.000000000 +0000
+++ ACE_wrappers/TAO/orbsvcs/orbsvcs/PortableGroup/PortableGroup_ORBInitializer.cpp	2007-02-13 11:45:38.000000000 +0000
@@ -14,7 +14,7 @@
 
 static const char pg_poa_factory_name[] = "TAO_GOA";
 static const char pg_poa_factory_directive[] = ACE_DYNAMIC_SERVICE_DIRECTIVE(
-  "TAO_GOA", "TAO_PortableGroup", "_make_TAO_PG_Object_Adapter_Factory", "");
+  "TAO_GOA", "TAO_PortableGroup-1.5.6", "_make_TAO_PG_Object_Adapter_Factory", "");
 
 TAO_BEGIN_VERSIONED_NAMESPACE_DECL
 
--- ACE_wrappers.orig/TAO/tao/ORB_Core.cpp	2007-01-25 19:59:59.000000000 +0000
+++ ACE_wrappers/TAO/tao/ORB_Core.cpp	2007-02-13 12:33:58.000000000 +0000
@@ -166,7 +166,7 @@
     poa_factory_directive_
       (ACE_TEXT_ALWAYS_CHAR
         (ACE_DYNAMIC_SERVICE_DIRECTIVE("TAO_Object_Adapter_Factory",
-                                       "TAO_PortableServer",
+                                       "TAO_PortableServer-1.5.6",
                                        "_make_TAO_Object_Adapter_Factory",
                                        ""))),
     alt_connection_timeout_hook_ (0)
@@ -1596,7 +1596,7 @@
     {
       this->configuration ()->process_directive (
         ACE_DYNAMIC_SERVICE_DIRECTIVE("PolicyFactory_Loader",
-                                      "TAO_PI",
+                                      "TAO_PI-1.5.6",
                                       "_make_TAO_PolicyFactory_Loader",
                                       ""));
       loader =
@@ -1635,7 +1635,7 @@
     {
       config->process_directive (
         ACE_DYNAMIC_SERVICE_DIRECTIVE ("ORBInitializer_Registry",
-                                       "TAO_PI",
+                                       "TAO_PI-1.5.6",
                                        "_make_ORBInitializer_Registry",
                                        ""));
       this->orbinitializer_registry_ =
@@ -2432,7 +2432,7 @@
     {
       this->configuration ()->process_directive
         (ACE_DYNAMIC_SERVICE_DIRECTIVE("TypeCodeFactory",
-                                       "TAO_TypeCodeFactory",
+                                       "TAO_TypeCodeFactory-1.5.6",
                                        "_make_TAO_TypeCodeFactory_Loader",
                                        ""));
       loader =
@@ -2465,7 +2465,7 @@
     {
       this->configuration()->process_directive
         (ACE_DYNAMIC_SERVICE_DIRECTIVE("CodecFactory",
-                                       "TAO_CodecFactory",
+                                       "TAO_CodecFactory-1.5.6",
                                        "_make_TAO_CodecFactory_Loader",
                                        ""));
       loader =
@@ -2492,7 +2492,7 @@
     {
       this->configuration()->process_directive
         (ACE_DYNAMIC_SERVICE_DIRECTIVE("Compression",
-                                       "TAO_Compression",
+                                       "TAO_Compression-1.5.6",
                                        "_make_TAO_Compression_Loader",
                                        ""));
       loader =
@@ -2519,7 +2519,7 @@
     {
       this->configuration()->process_directive
         (ACE_DYNAMIC_SERVICE_DIRECTIVE("TAO_POA_Current_Factory",
-                                       "TAO_PortableServer",
+                                       "TAO_PortableServer-1.5.6",
                                        "_make_TAO_POA_Current_Factory",
                                        ""));
       loader =
@@ -2549,7 +2549,7 @@
     {
       this->configuration ()->process_directive
         (ACE_DYNAMIC_SERVICE_DIRECTIVE("PICurrent_Loader",
-                                       "TAO_PI",
+                                       "TAO_PI-1.5.6",
                                        "_make_TAO_PICurrent_Loader",
                                        ""));
       loader =
@@ -2581,7 +2581,7 @@
     {
       this->configuration ()->process_directive
         (ACE_DYNAMIC_SERVICE_DIRECTIVE("DynamicAny_Loader",
-                                       "TAO_DynamicAny",
+                                       "TAO_DynamicAny-1.5.6",
                                        "_make_TAO_DynamicAny_Loader",
                                        ""));
       loader =
@@ -2609,7 +2609,7 @@
     {
       this->configuration()->process_directive
         (ACE_DYNAMIC_SERVICE_DIRECTIVE("IORManip_Loader",
-                                       "TAO_IORManip",
+                                       "TAO_IORManip-1.5.6",
                                        "_make_TAO_IORManip_Loader",
                                        ""));
       loader =
@@ -2636,7 +2636,7 @@
     {
       this->configuration ()->process_directive
         (ACE_DYNAMIC_SERVICE_DIRECTIVE("TAO_IORTable",
-                                       "TAO_IORTable",
+                                       "TAO_IORTable-1.5.6",
                                        "_make_TAO_Table_Adapter_Factory",
                                        ""));
       factory =
--- ACE_wrappers.orig/TAO/tao/ORBInitializer_Registry.cpp	2007-01-24 16:37:36.000000000 +0000
+++ ACE_wrappers/TAO/tao/ORBInitializer_Registry.cpp	2007-02-13 12:30:53.000000000 +0000
@@ -57,7 +57,7 @@
       {
         ACE_Service_Config::process_directive (
           ACE_DYNAMIC_SERVICE_DIRECTIVE("ORBInitializer_Registry",
-                                        "TAO_PI",
+                                        "TAO_PI-1.5.6",
                                         "_make_ORBInitializer_Registry",
                                         ""));
         orbinitializer_registry_ =
--- ACE_wrappers.orig/TAO/tao/PI/ORBInitInfo.cpp	2007-01-24 18:43:15.000000000 +0000
+++ ACE_wrappers/TAO/tao/PI/ORBInitInfo.cpp	2007-02-13 12:30:07.000000000 +0000
@@ -128,7 +128,7 @@
         {
           ACE_Service_Config::process_directive (
             ACE_DYNAMIC_SERVICE_DIRECTIVE("CodecFactory",
-                                          "TAO_CodecFactory",
+                                          "TAO_CodecFactory-1.5.6",
                                           "_make_TAO_CodecFactory_Loader",
                                           ""));
           loader =
--- ACE_wrappers.orig/TAO/tao/PortableServer/LifespanStrategyPersistent.cpp	2007-01-24 16:37:36.000000000 +0000
+++ ACE_wrappers/TAO/tao/PortableServer/LifespanStrategyPersistent.cpp	2007-02-13 11:46:59.000000000 +0000
@@ -85,7 +85,7 @@
             {
               ACE_Service_Config::process_directive (
                 ACE_DYNAMIC_SERVICE_DIRECTIVE(
-                  "ImR_Client_Adapter", "TAO_ImR_Client",
+                  "ImR_Client_Adapter", "TAO_ImR_Client-1.5.6",
                   "_make_ImR_Client_Adapter_Impl", ""));
 
               adapter =
--- ACE_wrappers.orig/TAO/tao/TAO_Internal.cpp	2006-12-28 11:46:39.000000000 +0000
+++ ACE_wrappers/TAO/tao/TAO_Internal.cpp	2007-02-13 11:47:57.000000000 +0000
@@ -484,7 +484,7 @@
             ACE_Service_Config::process_directive (
               ACE_DYNAMIC_SERVICE_DIRECTIVE (
                 "TAO_Codeset",
-                "TAO_Codeset",
+                "TAO_Codeset-1.5.6",
                 "_make_TAO_Codeset_Manager_Factory",
                 ""));
 
